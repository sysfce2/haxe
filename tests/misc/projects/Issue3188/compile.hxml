--macro Main.init()
-D message.reporting=pretty
-D message.no-color
