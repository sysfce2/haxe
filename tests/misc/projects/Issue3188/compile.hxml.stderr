[WARNING] external.txt:3: characters 5-8

 3 | 012345678
   |     ^^^
   | Correct line number

[WARNING] ./external.txt:3: characters 5-8

 3 | 012345678
   |     ^^^
   | Correct line number

